### ISSUE TITLE
WindowCommands - Toggle Button And Button Access Key Not Underlined When Pressing Alt

### ISSUE THREAD

**Issue by matjacques** [MahApps/MahApps.Metro#2746]
After the update from the version 1.2.4 to 1.3.0, the access key is no more underlined when pressing the Alt key. This only happens for buttons inside the RightWindowCommands and LeftWindowCommands. However, the action associated with the access key is correctly executed when the Alt+key combination is pressed. All buttons outside these zones properly display the underlined access key. 

For example:
```
    <Controls:MetroWindow.RightWindowCommands>
        <Controls:WindowCommands>
            <ToggleButton Content="_text" />
        </Controls:WindowCommands>        
    </Controls:MetroWindow.RightWindowCommands>
``` 
When pressing the Alt key, the 't' letter should be underlined.

Environment

- MahApps.Metro v1.3.0
- Windows 10
- Visual Studio 2015_
- .NET Framework 4.5

**Comment by punker76** [comment]
@matjacques The fix will be available in 1.4.0

**Comment by matjacques** [comment]
@punker76 Thanks! You're awesome :)
